examples/oneof_example_output.yml

```diff
@@ -14,6 +15,7 @@ components:
           round: "#/components/schemas/RoundShapeOptions"
 
     ShapeOptionsMergePatch:
+      description: "Options for a shape."
```

Comment:
I think that including this here is inconsistent with how we handle `BasicInterface`. In general, we probably don't want to pass the same description through to all variants. But we can talk about this later.